examples/interactive-containers/interactive-3d-slicer/custom_startup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# The content of /output is soft links to files in /input, hence copy will be skipped - so we delete the output dir first
```

Comment:
```suggestion
# 3D Slicer does not properly overwrite output files when they are soft links and not regular files.
# Since by default FCP sets up soft links from /output to /input, this causes a problem for 3D Slicer. To overcome this, we will delete the soft links in the /output directory and instead copy the relevant files from /input to /output as regular files.
```